apps/example-app/src/background.js

```diff
@@ -164,13 +163,12 @@ addEventListener("checkWatchReachability", (resolve, reject, args) => {
   }
 });
 
-addEventListener("sendMessageToWearable", (resolve, reject, args) => {
```

Comment:
The event listener function signature needs to be: `(resolve, reject, args) => void`